mkosi/config.py

```diff
@@ -4909,6 +4925,8 @@ def summary(config: Config) -> str:
       Kernel Modules Initrd Exclude: {line_join_list(config.kernel_modules_initrd_exclude)}
  Kernel Modules Initrd Include Host: {yes_no(config.kernel_modules_initrd_include_host)}
 
+                   Firmware Include: {line_join_list(config.firmware_include)}
+                   Firmware Exclude: {line_join_list(config.firmware_exclude)}
                              Locale: {none_to_default(config.locale)}
```

Comment:
There should be a newline between these two fields

Reply:
Sorry, I completely misunderstood https://github.com/systemd/mkosi/pull/3483#discussion_r1946223262. It should be ok now.